packages/react-query-devtools/package.json

```diff
@@ -12,7 +12,7 @@
   },
   "module": "build/lib/index.mjs",
   "main": "build/lib/index.js",
-  "browser": "build/umd/index.production.js",
+  "browser": "build/lib/index.js",
```

Comment:
This should allow for older bundlers (e.g. Webpack 4) to properly resolve `@tanstack/react-query-devtools`

Reply:
I would argue that we should remove `browser` field from all packages.
UMD build should be linked directly from CDN.
We do not ship separate code for `node` and `browser` (ie. xxx.server.js and xxx.client.js), so we could stick to `main` field.
Or remap all packages this way.